bin/main

```diff
@@ -1,2 +1,61 @@
 #!/usr/bin/env ruby
-puts 'Hello world!'
+require 'colorize'
+class GameUi
```

Comment:
The project is not working because you are requiring a gem that is not installed and is not in your Gemfile. Remove it or add it to the Gemfile and run bundle install